src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
+/* eslint-disable no-console */
+/* eslint-disable no-shadow */
```

Comment:
Consider removing 'eslint-disable no-shadow' and fix the code accordingly to avoid variable shadowing issues. Variable shadowing can cause confusion and make the code harder to understand.